lib/gzr/commands/role/cat.rb

```diff
@@ -40,9 +40,10 @@ def initialize(role_id,options)
         def execute(input: $stdin, output: $stdout)
           say_warning("options: #{@options.inspect}") if @options[:debug]
           with_session do
-            data = query_role(@role_id)
-            write_file(@options[:dir] ? "Role_#{data.id}_#{data.name}.json" : nil, @options[:dir], nil, output) do |f|
-              f.puts JSON.pretty_generate(data.to_attrs)
+            data = query_role(@role_id)&.to_attrs
```

Comment:
do you need a not found capture here or should the error just propagate?

Reply:
The error propagates. I tested that scenario.